app/Locale/Lang.php

```diff
@@ -2,17 +2,19 @@
 
 namespace App\Locale;
 
+use App\Contracts\Language;
+
 class Lang
 {
-	public static function get_classes()
+	private static function get_classes()
 	{
 		$return = [];
 		$list_lang = scandir(__DIR__);
 		for ($i = 0; $i < count($list_lang); $i++) {
-			if ($list_lang[$i] != '.' &&
+			if (
+				$list_lang[$i] != '.' &&
 				$list_lang[$i] != '..' &&
-				$list_lang[$i] != 'Lang.php' &&
-				$list_lang[$i] != 'LangInterface.php'
+				$list_lang[$i] != 'Lang.php'
```

Comment:
So _this_ is why the server craps out whenever I have a locale file open in `vi` (which creates a temporary file in the same directory named, e.g., `.English.php.swp`)?! I've always wondered, but never enough to investigate...

OK, so this question is unrelated to this PR, but is there a reason why we don't first filter out any name that doesn't end with the `.php` extension? Things would be a lot more robust then...

Reply:
sure. :)